ui/src/app/views/workspace/component.ts

```diff
@@ -81,6 +82,10 @@ export class WorkspaceComponent implements OnInit {
                 this.cohortList.push(coho);
               }
               this.cohortsLoading = false;
+            },
+            error => {
+              this.cohortsLoading = false;
```

Comment:
I don't believe so. I think it takes a function on next, on error, and on complete (where complete is a very specific notification an observable can get. I don't think it gets it if an error happens.)